### ISSUE TITLE
Testing-result

### ISSUE THREAD

**Issue by LinRui9531** [amdegroot/ssd.pytorch#363]
When i trained your ssd with voc dataset in my device (the cfg and parase were default), the loss was not convergent. 
![image](https://user-images.githubusercontent.com/28668008/58946676-068b4200-87b9-11e9-9cbe-8fcf7295790c.png)

And the result was  as followed: 
![image](https://user-images.githubusercontent.com/28668008/58946414-5e757900-87b8-11e9-97d0-364253fb5a76.png)
![image](https://user-images.githubusercontent.com/28668008/58946419-62090000-87b8-11e9-8cf6-61a4252ed176.png)

**Comment by LinRui9531** [comment]
I found the reason. I used python2.7.
In data/voc0712.py line 67-70. 
               `cur_pt = int(bbox.find(pt).text) - 1
                # scale height or width
                cur_pt = cur_pt / width if i % 2 == 0 else cur_pt / height`
the cur_pt , width and heght are all int.
cur_pt/(width or height) = [0,1]
repalce cur_pt or (width and height)  as float.
